packages-openshift.yaml

```diff
@@ -189,6 +189,32 @@ postprocess:
     ---
     EOF
 
+  # Inject symlink for Two Node OpenShift with Fencing (TNF)
+  # This is a workaround to allow pacemaker to use a pre-release resource agent for etcd
+  # This should be removed after https://issues.redhat.com/browse/OCPEDGE-1706 is completed
+  - |
+    #!/usr/bin/env bash
+    set -xeuo pipefail
+    rpm-ostree install --apply-live dnf-utils -y
+    set +e
+    python -m dnf.cli.main > /dev/null
+    if [ $? -ne 0 ]; then
+      exit 1
+    fi
+    python -m dnf.cli.main repoquery -l resource-agents | grep /usr/lib/ocf/resource.d/heartbeat/podman-etcd
```

Comment:
You should be able to do that with `rpm` directly?

Reply:
Maybe it's a knowledge gap thing, but I don't know where the RPM lives.
I know `rpm -q` can help for local RPMs, but how do I get the correct RPM in the first place?

The easiest way I could think to do that is to ask the tool for installing RPMs to look it up.

Reply:
Definitely don't want to install packages here.

Hmm, isn't this whole block basically

```bash
if ! test -f /usr/lib/ocf/resource.d/heartbeat/podman-etcd; then
    mkdir -p /usr/lib/ocf/resource.d/ocp-tnf
    ln -s /usr/local/lib/ocf/resource.d/ocp-tnf/podman-etcd /usr/lib/ocf/resource.d/ocp-tnf/podman-etcd
fi
```

?

But again, I would still prefer we add an e.g. `else` block there and `exit 1` so we know when to pull this out.

Reply:
That file won't be present until MCO enables the `two-node-ha` extension, so it would never be there by default in the rhcos base image.